bulbs/instant_articles/templates/instant_article/base_instant_article.html

```diff
@@ -38,7 +38,7 @@ <h3 class="op-kicker">
 
         <figure>
           {% block cover %}
-            <img src="{% cropped_url content.detail_image.id %}" />
+            <img src="{% cropped_url content.detail_image.id fixed=True %}" />
```

Comment:
Sweet!

On Fri, Jun 3, 2016 at 1:13 PM, Michael Butkovic <notifications@github.com>
wrote:

> In
> bulbs/instant_articles/templates/instant_article/base_instant_article.html
> <https://github.com/theonion/django-bulbs/pull/209#discussion_r65742981>:
>
> > @@ -38,7 +38,7 @@ <h3 class="op-kicker">
> >
> >          <figure>
> >            {% block cover %}
> > -            <img src="{% cropped_url content.detail_image.id %}" />
> > +            <img src="{% cropped_url content.detail_image.id fixed=True %}" />
>
> just added:
> https://trello.com/c/6dygMdoH/120-django-betty-cropper-rename-fixed-and-betty-fixed-url
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/theonion/django-bulbs/pull/209/files/90054b2101f00e3fa27534fce065f37fce2def2b#r65742981>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADAkUblvhln6fUV6QTZunRK4i3t7R2XOks5qIGCogaJpZM4IoAZK>
> .
>



-- 
Mike Parent
Developer
Onion, Inc.